docs/learn/learn-nominator.md

```diff
@@ -34,18 +35,22 @@ generate.
 
 While your DOT are staked by nominating a validator, they are 'locked' (bonded). You can receive new
 DOT in your account but you cannot stake as validator or transfer DOT away from your account. You
-can [un-nominate at any time](../maintain/maintain-guides-how-to-nominate-polkadot.md) to stop staking your
-funds. Keep in mind that the un-nomination is effective in the next era, and that un-nominating does
-not automatically unbond your funds. There is an unbonding period of 7 days on Kusama and 28 days on
-Polkadot before bonded funds can be transferred after issuing an unbond transaction.
+can [un-nominate at any time](../maintain/maintain-guides-how-to-nominate-polkadot.md) to stop
+staking your funds. Keep in mind that the un-nomination is effective in the next era, and that
+un-nominating does not automatically unbond your funds. There is an unbonding period of
+{{ polkadot: 28 days on Polkadot :polkadot }} {{ kusama: 7 days on Kusama :kusama }} before bonded
+funds can be transferred after issuing an unbond transaction. This is calculated by taking the
+`bonding duration` (in eras), multiplying it by the `length of a single era` (in hours), and
+dividing by the `hours in a day` (24). Example:
+({{ polkadot: 28 × 24 ÷ 24 = 28 days :polkadot }}{{ kusama: 28 × 6 ÷ 24 = 7 days :kusama }}).
 
 ### Active vs. Inactive Nomination
 
 When you go to the [Account actions](https://polkadot.js.org/apps/#/staking/actions) under staking
 page, you should see your bonded accounts and nomination status. If not, you can follow
 [this](../maintain/maintain-guides-how-to-nominate-polkadot.md) guide to configure it first. Your
-nominations will be effective in the next era; eras are roughly 6 hours on Kusama and 24 hours on
-Polkadot.
+nominations will be effective in the next era; eras are roughly
+{{ polkadot: 24 hours on Polkadot. :polkadot }}{{ kusama: 6 hours on Kusama. :kusama }}
```

Comment:
I think additional math is required to get this value on-chain from `babe.epochDuration` and `staking.sessionsPerEra`.  Another potential use-case for having an RPC math component.

Reply:
Makes sense.